src/model/serieslistmodel.cpp

```diff
@@ -212,18 +222,6 @@ void SeriesListModel::updateAllSeries(bool updateEndedSeries)
     m_episodes.clear();
     m_banners.clear();
 
-    auto allSeries = m_dbmanager->getSeries();
-    for (auto series : allSeries) {
-        if (!updateEndedSeries) {
-            if (series["status"] != "Ended") {
-               m_seriesIds.append(series["id"].toString());
-            }
-        } else {
-             m_seriesIds.append(series["id"].toString());
-        }
-    }
-
-    qDebug() << "m_seriesIds" << m_seriesIds;
-
+    m_seriesIds = m_dbmanager->getSeriesIds(updateEndedSeries);
```

Comment:
Protect the private members somehow, now since it is finally possible to do update / adding at the same these will get reused at the wrong time :|

Reply:
One idea is to store to db once something arrives and not wait everything to be ready :thinking:

Reply:
done :+1: